packages/rspack/package.json

```diff
@@ -60,5 +60,8 @@
   },
   "nx-migrations": {
     "migrations": "./migrations.json"
+  },
+  "publishConfig": {
```

Comment:
I thought we needed this for provenance... how does rspack have provenance without this?

Reply:
It's not for provenance, it's needed because the default access for new scoped package is restricted and so the first time we publish a new package we need this explicit config. Technically after that point we no longer need it anymore, but it doesn't do any harm, just makes its state explicit, and so having this be a rule for all package.json files means we won't run into any issues when new packages are added in future.